### PR TITLE
Add support for running the tests with tox

### PR DESCRIPTION
**Pull request by pypingou**
Signed-off-by: Pierre-Yves Chibon <pingou@pingoured.fr>

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+usedevelop = True
+deps =
+   pytest
+   coverage
+setenv =
+    PYTHONPATH={toxinidir}
+commands =
+    coverage run --source rdbsync -m py.test
+    coverage xml
+
```
Comment (bstinsonmhk) [inline tox.ini:16]: We need: 
`passenv = HOME` 

somewhere in here so we can find our directory when running tests in openshift

**Comment by bstinsonmhk** [comment]
:+1:

**Review by cverna — APPROVED** [review]
:+1:

**Comment by pypingou** [comment]
Thanks :)

**Pull request merged**